Filter/CallbackFilter.php

```diff
@@ -18,14 +18,6 @@ class CallbackFilter extends Filter
     /**
      * {@inheritdoc}
      */
-    protected function association(ProxyQueryInterface $queryBuilder, $data)
```

Comment:
I don't get where is the BC break. This method is defined in the [`Filter`](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/master/Filter/Filter.php#L37) class. That was only the overloading of this method that was removed.